builder/amazon/common/access_config.go

```diff
@@ -86,3 +100,24 @@ func (c *AccessConfig) Prepare(t *packer.ConfigTemplate) []error {
 
 	return nil
 }
+
+func GetInstanceMetaData(path string) (contents []byte, err error) {
+	url := "http://169.254.169.254/latest/meta-data/" + path
```

Comment:
https://github.com/awslabs/aws-sdk-go/pull/192 appears to be the pull request which needs tracking for now.